### DISCUSSION TITLE
General release date for 0.150.0

### DISCUSSION THREAD

**Discussion by jandersen-plaid** [helmfile/helmfile#622] [Questions] [answered]
Hiya! First off, really thankful that helmfile exists and that there is so much wonderful community support and maintenance work put towards the project. It has really improved the quality of life on our team for managing helm configurations.

This is really only one question: is there a release date or general timeline for the release date of 0.150.0?

Due to legacy reasons, the change in YAML templating introduced a change to our helmfile deployments that has made a certain chart unable to be deployed without causing a significant drop in traffic for our production services. A leading newline is consistently stripped by `goccy` which changes a configmap entry: https://github.com/goccy/go-yaml/issues/285. This is way more than a little bit ridiculous for us -- that this kind of issue can occur and cause a drop -- and we are migrating away from the situation that caused us to get in this position, but it will take some time.

Due to this bug, we are anxiously awaiting the release of 0.150.0 which would include https://github.com/helmfile/helmfile/pull/609 and the preceding patches that revert the change in YAML behavior (and make it optional going forward). We can and will create our own release pipeline // build that includes the patch (since, thankfully, building helmfile is super simple and accessible!), but we just want to get a quick glance at any potential release date to see if we can save some time duplicating effort. So, with that in mind, if there is not timeline or date, that is cool too! We will just build the project with the patch in it.

**Comment by yxxhero** [comment, marked as answer]
@jandersen-plaid thanks for using helmfile. Maybe v0.150.0 will be released next week.  
https://github.com/helmfile/helmfile/pull/613
https://github.com/helmfile/helmfile/pull/612
https://github.com/helmfile/helmfile/pull/602

**Comment by jandersen-plaid** [comment]
Fantastic, thank you so much @yxxhero!

**Comment by yxxhero** [comment]
@jandersen-plaid https://github.com/helmfile/helmfile/releases/tag/v0.150.0 please enjoy it. if you have other issues please give me feedback.
